src/zap/zapcode.cpp

```diff
@@ -1202,6 +1202,32 @@ void ZapUnwindData::Save(ZapWriter * pZapWriter)
 #endif //REDHAWK
 }
 
+#elif defined(_TARGET_X86_) && defined(FEATURE_PAL)
+
+UINT ZapUnwindData::GetAlignment()
+{
+    return sizeof(ULONG);
```

Comment:
Can this be just sizeof(BYTE) ? The JIT case does not seem to have any extra alignment. It can be same here.